http/css/style.css

```diff
@@ -20,12 +20,17 @@
 body {
     font-size: 85%;
     font-family: sans-serif;
-    background-color: #ffffff;
     display: flex;
     justify-content: center;
+    background: url("../images/background.png") left / cover no-repeat fixed;
```

Comment:
Nope, not doing it for me. I think the image brings a bit of character to the page.